src/crimson/os/seastore/cached_extent.h

```diff
@@ -181,8 +278,16 @@ class CachedExtent : public boost::intrusive_ref_counter<
    * references in the the buffer with the passed
    * record_block_offset record location.
    */
-  virtual void on_delta_write(paddr_t record_block_offset) {}
+  void on_delta_write(paddr_t record_block_offset) {
+    on_delta_commit(record_block_offset);
+    touched_by = 0;
+  }
+
+  virtual void on_delta_commit(paddr_t record_block_offset) {}
 
+  virtual void on_replace_prior(Transaction &t) {}
```

Comment:
This needs a comment explaining precisely what the implementation is meant to do.